packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/services/forms/FormWithCheckboxDescriptionProvider.java

```diff
@@ -88,6 +88,13 @@ private View createView() {
 
     private FormDescription createFormDescription() {
         var defaultStyle = new FormBuilders().newCheckboxDescriptionStyle()
+                .gridTemplateColumns("none")
```

Comment:
Do not add new responsibilities to existing code. Create dedicated test descriptions for your need. If you want to test the grid layout, create a description for that.

Is that done to artificially grow the code coverage?

Reply:
Yes this test has been done for coverage purpose.
To be coherent I will add assertion in the existing test that uses this Provider.
There is no need to create a new test with a new provider because it just adds the grid part without changing the existing test. In the test, I will check that the result of the rendering gives the same value that given in the view builder